README.md

```diff
@@ -21,26 +20,55 @@ Panther is a security analytics platform built for cloud-focused security teams.
 
 Panther enables teams to define detections as code and programmatically upload them to your Panther deployment.
 
-This repository contains all of Panther's built-in detections  installed by default.
+This repository contains all of Panther created detections that can be added to your Panther instance.
```

Comment:
maybe remove of? I can't tell if it is just me but this reads a little strange

Reply:
See updated comments